### PR TITLE
Bower support

### PR DESCRIPTION
**Pull request by kasiriveni**
adding the bower support

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "color",
+    "colorscheme",
+    "color",
+    "swatches",
+    "SCSS",
+    "LESS",
+    "Stylus",
+    "CSS"
+  ],
+  "homepage": "https://github.com/yeun/open-color",
```
Comment (darlanmendonca) [inline bower.json]: maybe be better change homepage to https://yeun.github.io/open-color/

**Comment by yeun** [comment]
Thanks for work @kasiriveni. :)
@sairion Could you check this pr?

**Comment by sairion** [comment]
Seems good.

**Comment by darlanmendonca** [comment]
It is necessary to make a new release (tag), to bower package works, because the last version 1.4.0 dont contains the bower.json

**Pull request merged**